app/src/main/java/ca/marshallasch/veil/ListAdapter.java

```diff
@@ -0,0 +1,63 @@
+package ca.marshallasch.veil;
+
+
+import android.app.Activity;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+
+public class ListAdapter extends RecyclerView.Adapter<ListAdapter.ViewHolder> {
+    private String[] titles;
+    private String[] content;
```

Comment:
Also please add some comments to this class with what it does. since it is not s straight forward class.

Reply:
Comments added